src/app/src/components/FacilityListItemsConfirmationTableRowItem.jsx

```diff
@@ -0,0 +1,124 @@
+                />
+            </TableCell>
+            <TableCell
+                padding="default"
+                style={listTableCellStyles.addressCellStyles}
+            >
+                <CellElement item={address} />
+            </TableCell>
+            <TableCell
+                padding="default"
+                colSpan={2}
+                variant="head"
+                style={{
+                    ...listTableCellStyles.headerCellStyles,
+                    textAlign: 'left',
+                }}
+            >
+                <div>
+                    <ShowOnly when={!readOnly}>
+                        {status !== facilityMatchStatusChoicesEnum.PENDING ? (
+                            <div
+                                key={id}
+                                style={
+                                    confirmRejectMatchRowStyles.cellRowStyles
+                                }
+                            >
+                                <div
+                                    style={
+                                        confirmRejectMatchRowStyles.cellActionStyles
+                                    }
+                                >
+                                    <div>{status}</div>
+                                </div>
+                            </div>
+                        ) : (
+                            <Checkbox
+                                onChange={() =>
+                                    toggleCheckbox({
+                                        id,
+                                        os_id,
+                                        address,
+                                        name,
+                                        confidence,
+                                    })
+                                }
+                                checked={
+                                    action === CONFIRM_ACTION
+                                        ? activeCheckboxes[0]?.id === id // eslint-disable-line camelcase
+                                        : activeCheckboxes.some(
+                                              activeItem =>
+                                                  activeItem.id === id, // eslint-disable-line camelcase
+                                          )
+                                }
+                                disabled={isCheckboxDisabled(id)}
+                            />
+                        )}
+                    </ShowOnly>
+                </div>
+            </TableCell>
+            <TableCell
+                padding="default"
+                variant="head"
+                style={{
+                    ...listTableCellStyles.headerCellStyles,
+                    textAlign: 'right',
+                }}
+            >
+                <b>{confidence}</b>
+            </TableCell>
+        </TableRow>
+    );
+}
+
+export default FacilityListItemsConfirmationTableRowItem;
```

Comment:
Could you please use named export?

Default exports don’t associate any name with the item being exported, meaning that any name can be applied during import. This may give you a flexibility, but if multiple developers imports the same module with different names or even non descriptive name, we are screwed.
Named exports are explicit, forcing the consumer to import with the names the original author intended and removing any ambiguity.

E.g.
```
// ❌
export default MyComponent;

// ✅
export { MyComponent };
```

Reply:
Ok, but consider the whole codebase in the FE has this problem.

Reply:
Upd: there is a lint rule that forces to use default exports:
> error  Prefer default export  import/prefer-default-export

This can be fixed in one of the tech debt tickets.

Reply:
I see. Thanks!